rust/navigator/src/actions.rs

```diff
@@ -24,6 +24,7 @@ pub enum Action {
     NetworkSelector,
     NextUnit,
     PreviousUnit,
+    ChangeNetwork,
```

Comment:
That is probably a bad merge; was removed in previous PRs

Reply:
This is intentional.
The previous implementation of the derivation import changed the network navstate, but the new one does not. However, I need a way to mimic the old behavior for `flow_test_1`, so `ChangeNetwork` is very useful for this purpose.